examples/complete_parser.cpp

```diff
@@ -54,97 +54,97 @@ int main(int argc, char* argv[])
         petro::box::ftyp,
         petro::box::pdin,
         petro::box::moov<petro::parser<
-            petro::box::mvhd,
-            petro::box::trak<petro::parser<
-                petro::box::tkhd,
-                petro::box::tref,
-                petro::box::edts<petro::parser<
-                    petro::box::elst
-                >>,
-                petro::box::mdia<petro::parser<
-                    petro::box::mdhd,
-                    petro::box::hdlr,
-                    petro::box::minf<petro::parser<
-                        petro::box::vmhd,
-                        petro::box::smhd,
-                        petro::box::hmhd,
-                        petro::box::nmhd,
-                        petro::box::dinf<petro::parser<
-                            petro::box::dref<petro::parser<
-                                petro::box::url,
-                                petro::box::urn
-                            >>
+                petro::box::mvhd,
```

Comment:
That is not an option. I could suppress the indent for the first opening <, but how should astyle decide to apply the dedent if it sees the closing > and the second >, but there is no corresponding indent for one of them? This would be a total mess.
I think that the current solution is very logical. You opened 2 levels of template instantiations, so you add 2 levels of indentation for that.